modules/sql/src/main/resources/db/migration/V060__Light_Sinks.sql

```diff
@@ -0,0 +1,29 @@
+---------------------------
+-- Science Fold Light Sinks
+-- The science fold directs the light to one of this named positions
+---------------------------
+
+CREATE TABLE e_light_sink_names (
+    id identifier  PRIMARY KEY NOT NULL ,
+    name VARCHAR(15)    NOT NULL CHECK (name ~ '[a-z][a-z0-9]*')
```

Comment:
if name is unique perhaps we don't need the `id`? @tpolecat @swalker2m WDYT?

Reply:
You need the `id` to map to/from the enumerated type.  I suppose you wouldn't have to have a `name` but I assume the name is changeable and/or doesn't conform to the identifier convention.  In fact, the convention for other enums is to have a `short_name` *and* a descriptive `long_name`.

Reply:
In this case, the name does not conform to identifier conventions.